src/frontend/apps/desk/cunningham.ts

```diff
@@ -265,21 +265,22 @@ const config = {
           'border-radius': '0',
```

Comment:
might be great to keep Johann updated with these changes, so he can update design.